packages/runtime/container-runtime/src/garbageCollection.ts

```diff
@@ -1408,6 +1420,7 @@ export class GarbageCollector implements IGarbageCollector {
 					isSummarizerClient: this.isSummarizerClient,
 					url: trimLeadingSlashes(toNodePath),
 					nodeType,
+					gcEnforcementDisabled: false,
```

Comment:
False gcEnforcementDisabled means it's enabled? Why is it always enabled here? And shouldn't this be true in a sense since tombstoneEnforcement is not true?

Reply:
I think this is leftover from when I was also covering Sweep.  Good catch, I'll remove.

Reply:
Umm wrong, I need to pass in the flag from ContainerRuntime into here